src/webui/www/private/confirmtrackerdeletion.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8">
+    <title>QBT_TR(Remove tracker)QBT_TR[CONTEXT=confirmDeletionDlg]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css">
+    <script src="scripts/lib/MooTools-Core-1.6.0-compat-compressed.js"></script>
+    <script src="scripts/lib/MooTools-More-1.6.0-compat-compressed.js"></script>
+    <script>
+        "use strict";
+
+        window.addEventListener("DOMContentLoaded", () => {
+            const url = new URI().getData("url");
+            $("confirmDeleteTrackerText").innerHTML = 'QBT_TR(Are you sure you want to remove tracker "%1" from all torrents?)QBT_TR[CONTEXT=TrackersFilterWidget]'.replace("%1", window.parent.qBittorrent.Misc.escapeHtml(decodeURIComponent(url)));
```

Comment:
You can sidestep the 'quote escaping' issue by reordering the code, consider this:
```suggestion
            $("confirmDeleteTrackerText").textContent = 'QBT_TR(Are you sure you want to remove tracker %1 from all torrents?)QBT_TR[CONTEXT=TrackersFilterWidget]'.replace("%1", `"${window.parent.qBittorrent.Misc.escapeHtml(decodeURIComponent(url))}"`);
```

Reply:
The downside of this is that the string will no longer match the one used in the GUI and thus will require a distinct translation.

Reply:
>The downside of this is that the string will no longer match the one used in the GUI and thus will require a distinct translation.

I don't see it a big problem. 
BTW I'm going to get rid of `innerHTML` in a upcoming PR so it will be changed anyway.

Reply:
WebUI has separate translations.

Reply:
Doesn't the Web UI use the same translations in the case where the `CONTEXT` points to a GUI class?

Reply:
>Doesn't the Web UI use the same translations in the case where the CONTEXT points to a GUI class?

no.

Reply:
> Doesn't the Web UI use the same translations in the case where the `CONTEXT` points to a GUI class?

It doesn't point to GUI class, it just has similar name.

Reply:
This is news to me!

I'll update the string to move the quotes and switch to `textContent`.